jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -774,8 +774,8 @@ int capacityInternal() {
  * @param 		index	the zero-based index in this StringBuffer
  * @return		the character at the index
  *
- * @exception	IndexOutOfBoundsException when <code>index < 0</code> or
- *				<code>index >= length()</code>
+ * @exception	IndexOutOfBoundsException
+ *              If {@code index} is negative or {@code index} is grater or equal length()
```

Comment:
typo in grater - greater

Reply:
Some how I missed this line.. actually this should be fixed with this line,
when {@code index < 0} or {@code index >= length()}